script/Arcueid/Day 5/05_00_ARC05_3_2B.txt

```diff
@@ -207,7 +207,8 @@ It would be futile to hold him down directly. Besides, I've never done it before
 [sha:7371bf30d3635da3426e39418d57415e9e5c5622]{
 -- Page 262, Offset 9241.
 -- 「鎖……？　鎖って、あの鎖か？」
-Chains? You mean like %{i}metal%{/i} chains?
+// can the lint be set to detect if both quotes are missing in voiced lines?
```

Comment:
It's pretty dumb rn, I can probably make it smarter by examining the JP line for quote markers